concent_api/utils/helpers.py

```diff
@@ -166,15 +169,27 @@ def get_storage_source_file_path(subtask_id, task_id):
     return get_storage_file_path('source', subtask_id, task_id)
 
 
-def calculate_maximum_download_time(size: int) -> int:
+def calculate_maximum_download_time(size: int, rate: int = constants.DEFAULT_UPLOAD_RATE) -> int:
```

Comment:
Please move it out of `utils`. This app is meant for much more generic stuff. This is part of our business logic, not a general purpose utility and belongs in `core`.

Reply:
:+1: